troubleshooting/troubleshooting-vse-path-too-long-exception.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Path Too Long Exception when Building Project in Microsoft Visual Studio
+description: AppBuilder Projects with fully qualified file names longer that 260 symbols fail to build in Visual Studio Extension.
```

Comment:
* There's no need for Projects to be capitalized here.
* Use "in the extension for Visual Studio" instead of "in Visual Studio Extension". Note that extension does not need to be capitalized here as it is not a proper name.